api/config/config.example.json

```diff
@@ -1,33 +0,0 @@
-{
-  "discordApiKeys": {
-    "ENABLED": true,
-    "CLIENT_ID": "NOT_SET",
-    "CLIENT_SECRET": "NOT_SET",
-    "REFRESH_TOKEN": "NOT_SET"
-  },
-  "googleApiKeys": {
-    "ENABLED": true,
```

Comment:
add this file back, be sure it looks exactly the same as this file and has a blank line at the very bottom